packages/theme/src/cli/utilities/theme-store.ts

```diff
@@ -3,7 +3,10 @@ import {store as conf, error} from '@shopify/cli-kit'
 export function getTheme(flags: {store: string | undefined}): string {
   const store = flags.store || conf.cliKitStore().getTheme()
   if (!store) {
-    throw new error.Abort('A store is required', 'Specify the store using --store={your_store}')
+    throw new error.Abort(
+      'A store is required',
+      'Specify the store passing --store={your_store_url} or set the SHOPIFY_FLAG_STORE environment variable.',
```

Comment:
Since we have the flags extracted in a module, what about reading the flag name and the environment variable from the flag?

Reply:
Done!